src/main/resources/youtube/settings/host/values/strings.xml

```diff
@@ -1840,8 +1839,7 @@ Tap on the continue button and disable battery optimizations."</string>
 • Stable volume is not available."</string>
     <string name="revanced_spoof_streaming_data_side_effects_unknown">• Video may not play.</string>
     <string name="revanced_spoof_streaming_data_ios_compatibility_title">iOS Compatibility mode</string>
-    <string name="revanced_spoof_streaming_data_ios_compatibility_summary_on">Only spoofed as iOS client if it is not a movie, paid video, or livestream.</string>
-    <string name="revanced_spoof_streaming_data_ios_compatibility_summary_off">Always spoofed as iOS client.</string>
+    <string name="revanced_spoof_streaming_data_ios_compatibility_summary">Check if the video is a movie or a members-only video to prevent unplayable errors.</string>
```

Comment:
Before merging PR
--
- iOS Compatibility mode ON:
1. spoofed as Android VR client or Android TV client for livestreams and paid videos
2. spoofed as iOS client for other videos

- iOS Compatibility mode OFF:
1. always spoofed as iOS client

After merging PR
--
- iOS Compatibility mode ON
1. spoofed as Android VR client or Android TV client for livestreams and paid videos
2. spoofed as iOS client for other videos
- iOS Compatibility mode OFF:
1. spoofed as Android VR client or Android TV client for paid videos
2. spoofed as iOS client for other videos

I still think the intent of the PR is unclear

Reply:
Uhm I see. Currently there are 3 ways to keep this PRs:
- Or keep this PRs like now
- Or find a way to check paid video + movie without needing additional fetch
- Or completely remove iOS Compatibility Mode because it seems that paid videos + movies only affects a very small number of users

Reply:
It was very useful in my case, where I didn't have a way to check if the video was Live.

Maybe it would be a better solution than currently one used by Inotia to perform this same verification.